care/emr/migrations/0023_allergyintolerance_allergy_intolerance_type.py

```
@@ -0,0 +1,18 @@
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('emr', '0022_facilitylocation_sort_index'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='allergyintolerance',
+            name='allergy_intolerance_type',
+            field=models.CharField(default='allergy', max_length=20),
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)